lib/resources/example_json/r4_examples_organization.rb

```diff
@@ -213,5 +213,153 @@ module Resources
         }
       ]
     }.freeze
+
+    R4_ORGANIZATION_CAREGIVER_SEARCH_BUNDLE ||= {
+      'resourceType': 'Bundle',
+      'id': '9cdc9b0a-1f7f-4a33-8c57-76514a7c7cdb',
+      'type': 'searchset',
+      'total': 2,
+      'link': [
+        {
+          'relation': 'self',
+          'url': 'http://localhost:3000/r4/5a12b3e8-5941-4076-8592-1fb463441c68/Organization/$get-cg-for-mrcu?_id=3172822'
```

Comment:
can we have a better example than the URL being 'localhost?

Reply:
Updated here: dc955510baaba090a3292040cfd13ddeedcf5e04